darwin/cli_functions.py

```diff
@@ -150,6 +150,20 @@ def set_team(team_slug: str) -> None:
     config = _config()
     config.set_default_team(team_slug)
 
+def set_compression_level(compression_level: int) -> None:
+    """
+    Change the compression level of text/json contents sent to backend and persist the change on the configuration file.
```

Comment:
`backend` is not the best word considering this is public message. How about "Darwin APIs"?